commnad_line_rock_paper_scissors.py

```diff
@@ -0,0 +1,36 @@
+""" Object-Oriented Rock Paper Scissors """
+
+import random
+
+from rock_paper_scissors.user import User
+
+from rock_paper_scissors.game import Game
+
+
+def main():
+
```

Comment:
PEP8 convention [advises](https://www.python.org/dev/peps/pep-0008/#blank-lines) against separating a function definition and the first functional line with a blank line, and this is further [backed up](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) by PEP 257.

Should be:
```python
def main():
    human = User('Human')
```